zio/zeekio/parser.go

```diff
@@ -85,6 +86,7 @@ func (p *Parser) parseTypes(types []string) error {
 }
 
 func (p *Parser) ParseDirective(line []byte) error {
+	line = bytes.TrimSpace(line)
```

Comment:
nit: its not explicitly spelled out in the spec but all the examples show no space between the # and the actual directive, but this will allow for space there.  It appears to me that the leading # has already been stripped when we get here despite the presence of the next line.  I guess putting those two points together this would accept something like
```
#   #types ...
```

Reply:
Good catch, this is not just a nit. The leading # has been stripped when we get here from zngio.Reader, but not when we get here from elsewhere. I'll adjust the latter and remove the TrimSpace from here.